kms-secrets.develop.us-west-2.yml

```diff
@@ -0,0 +1,3 @@
+secrets:
+    SECRETS: AQICAHjHkVxHY8dk4wKV7JyoKushbE+j3zsd9T+POPjRjk4kBAHmagmEpEBq/FyV7LkkQcw0AAACIjCCAh4GCSqGSIb3DQEHBqCCAg8wggILAgEAMIICBAYJKoZIhvcNAQcBMB4GCWCGSAFlAwQBLjARBAxp4Bx7YTFfeiR05hoCARCAggHV6fcPE4+U1WmrHjKjl8jYtl+mKAcU2SxUhS2Kkv8tlK6ZNVg4D/LEvDi4UcDe0c2rAp2X45DdtmtIr0efTNR5WG3KRr169xeJuWDAo2jWZL8D7D70DEbw09R6Bj1BMnnLJ2YiAsz8d2unZIHeVnAD3gN9YbcCSqYAnV0Q7GUu0quo6lMVNsP6FwHQAO+cWolGIgwPXBPDi+5YKbNjUHCvd4WVcq4l2KyzhvuWtgDh77ZG0b4Ns+Fttq2WCYLXAcvcOJ1X9Waqp1DW52Hxz3R/fqNhffWOJMAtPba8/6XFAIZVxNvd8atuoLsfynPzFfXR04DyB4o5yKNMr1RNr9a3mqZVW2XA/bhw4P6XMPnEuQhFv2ytPJSYQgQMspvXEm/Z/52b3L1XuK2eI5tu4uJSTLOG875UyNsnTBs/lvZKoS9FR0EeYkCbd801YwfK0oNcbA5a3cgg386DatheKKLNnYee0PpOkA9iMJ7Sjg38tUGxUcdk5VecZjPZKqHjN3dAiTU7GbUDcAjJgtgRSR37JqlbTwRJ3fmQn4+fOShwO24s0FiRdQkQObIyMTGTq2OsOLcmD//wDRKwqzsZtTzExwO0eRIDryDOP6/9g/0gT2TF6YPwxg==
```

Comment:
What is this, and should it live in the github repo?

Reply:
It's the content of the secrets encrypted. You in order to decrypt them you need to get access to that key, and for that you need:
1) An AWS account
2) Permission to read keys
3) Permission to read the resource

Sure we can use vault or another secrets repository, but this solution is the fastest way to test our services locally and at the same time, it ensures it will work on lambda.
Not bulletproof but safe enough